test/features/MyFirstApiConsume.feature

```diff
@@ -0,0 +1,36 @@
+Feature: My First Api Consume
+    Scenario: Verify GET Service
+        Given a REST API url
+        When we consume a GET Service
```

Comment:
If you know who is consuming the service it makes the scenario more valuable, as it tells you what is the expected behavior for a user, this could be something like

`When an unauthorized user consumes a GET service`

Reply:
same thing for the rest `we`